### PR TITLE
Bridge UI Improvement 2

### PR DESCRIPTION
**Pull request by hungdoansy**
1. Show a tooltip if user receives anyToken
image

2. Re-style Bridge form

| Before | After |
| --- | --- |
| image | image |

4. Add a Guide button

5. Add Mixpanel events

6. Update mobile layout of the history table

| Before | After | 
| --- | --- |
| after_mobile_bridge_history | before_mobile_bridge_history |

7. Update formatter of token receive amount. Below is how the worst case looks like

| Before | After |
| --- | --- |
|  before_formatter_bridge | after_formatter_bridge |

### CODE REVIEW

File: src/hooks/useMixpanel.ts [hunk 1]
```diff
@@ -834,6 +836,27 @@ export default function useMixpanel(currencies?: { [field in Field]?: Currency }
           mixpanel.track('Bridge - User click to Subscribe button')
           break
         }
+        case MIXPANEL_TYPE.BRIDGE_CLICK_DISCLAIMER: {
+          if (typeof payload !== 'boolean') {
+            throw new Error(`Wrong payload type for Mixpanel event: ${MIXPANEL_TYPE.BRIDGE_CLICK_DISCLAIMER}`)
```
Comment (nguyenhoaidanh) [inline src/hooks/useMixpanel.ts:841]: will crash or not, double check

File: src/hooks/useMixpanel.ts [hunk 2]
```diff
@@ -834,6 +836,27 @@ export default function useMixpanel(currencies?: { [field in Field]?: Currency }
...
+          }
+
+          mixpanel.track('Bridge - User click to Checkbox Disclaimer in Confirmation popup', {
+            platform: isMobile ? 'Mobile' : 'Web',
+            network,
```
Comment (nguyenhoaidanh) [inline src/hooks/useMixpanel.ts:846]: remove all , already tracked by default https://github.com/KyberNetwork/dmm-interface/blob/46f5c2bb6b84ac75947b76827a3248c14c661005/src/hooks/useMixpanel.ts#L245

File: src/hooks/useMixpanel.ts (hunk 1 above)
Comment (hungdoansy) [reply src/hooks/useMixpanel.ts:841]: Will crash in dev mode only. This works like a type hint for what payload should be

File: src/hooks/index.ts [hunk 3]
```diff
@@ -70,8 +72,10 @@ export function useActiveWeb3React(): {
         : undefined,
     [isConnectedSolana, connectedWalletSolana?.adapter],
   )
-  const mockAccountEVM = ''
-  const mockAccountSolana = ''
+
+  const accountFromQs = [qs['account']].flat()[0]
```
Comment (namgold) [inline src/hooks/index.ts:76]: j z tr

Comment (hungdoansy) [reply src/hooks/index.ts:76]: This is a mock. Will be removed after testing's done. `qs['account']` can be string or string[] so I wrap it inside an array and flat it. `accountFromQs` is of type string

File: src/pages/Bridge/BridgeTransferHistory/TokenReceiveCell.tsx [hunk 4]
```diff
@@ -1,24 +1,71 @@
...
+import { useMedia } from 'react-use'
+import { Flex, Text } from 'rebass'
 
+import { MouseoverTooltip } from 'components/Tooltip'
 import { MultichainTransfer } from 'hooks/bridge/useGetBridgeTransfers'
+import useTheme from 'hooks/useTheme'
 import { formatAmountBridge } from 'pages/Bridge/helpers'
+import { ExternalLink, MEDIA_WIDTHS } from 'theme'
+
+const formatAmountBridgeForMobile = (rawAmount: string | number) => {
```
Comment (nguyenhoaidanh) [inline src/pages/Bridge/BridgeTransferHistory/TokenReceiveCell.tsx:11]: let's add some examples for this function
ex: 1123.232323 =>  ...

File: src/hooks/index.ts (hunk 3 above)
Comment (nguyenhoaidanh) [reply src/hooks/index.ts:76]: `String(qs.account || '')`

Comment (nguyenhoaidanh) [reply src/hooks/index.ts:76]: I'm ok with keeping this code after testing. 🚀

**Review by nguyenhoaidanh — APPROVED** [review]

File: src/pages/Bridge/ComfirmBridgeModal.tsx [hunk 5]
```diff
@@ -116,6 +117,13 @@ export default memo(function ConfirmBridgeModal({
   const [accepted, setAccepted] = useState(false)
   const { account, chainId } = useActiveWeb3React()
   const [{ chainIdOut, currencyIn, currencyOut }] = useBridgeState()
+  const { mixpanelHandler } = useMixpanel()
+
+  const handleClickDisclaimer = useCallback(() => {
+    const newValue = !accepted
+    setAccepted(newValue)
```
Comment (viet-nv) [inline src/pages/Bridge/ComfirmBridgeModal.tsx:124]: `setAccept(prev => !prev)`

**Pull request merged**